ansible/roles/schulcloud-server-core/templates/api-fwu-svc.yml.j2

```diff
@@ -5,9 +5,6 @@ metadata:
   namespace: {{ NAMESPACE }}
   labels:
     app: api-fwu
-  annotations:
-    prometheus.io/scrape: 'true'
-    prometheus.io/port: '4446'
```

Comment:
It's fine, but please make you an follow up ticket where you add the new metric endpoint to this service.